packages/web_drivers/README.md

```diff
@@ -1,9 +1,18 @@
 ```
-# Chrome Driver versions are stored in driver_lock.yaml
 # To install and start Chrome Driver.
 
 dart lib/web_driver_installer.dart chromedriver &
 
+# Chrome Driver versions are stored in driver_lock.yaml. Driver version can also
```

Comment:
Did you mean `driver_version.yaml`?

Reply:
yes! it used to be called lock back in the day.